### PR TITLE
Adding SAGA recipe

### PR DESCRIPTION
**Pull request by iparask**
This is a recipe to inlcude SAGA-python in conda-forge.

### CODE REVIEW

File: recipes/saga-python/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed  .
+  skip: True  # [win]
+  skip: True  # [py3k]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```
Comment (jakirkham) [inline recipes/saga-python/meta.yaml]: Does it compile code? If not, we don't need this.

File: recipes/saga-python/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed  .
+  skip: True  # [win]
+  skip: True  # [py3k]
```
Comment (jakirkham) [inline recipes/saga-python/meta.yaml]: Can fuse these into `skip: True  # [py3k or win]`.

**Comment by iparask** [comment]
@andre-merzky Do you agree to be a maintainer here?

**Comment by andre-merzky** [comment]
yes, thanks for checking.

**Pull request merged**